README.md

```diff
@@ -1,32 +1,69 @@
-# emptyLog
+# empty
 
-Workflow to log n lines from a file and exit as specified.
+Workflow for testing infrastructure
 
 ## Overview
 
 ## Dependencies
--
+
+
 
 ## Usage
 
 ### Cromwell
-
 ```
-java -jar cromwell.jar run empty --inputs inputs.json
+java -jar cromwell.jar run empty.wdl --inputs inputs.json
 ```
 
 ### Inputs
 
-#### Required Workflow Parameters:
-Parameter | Type | Description
----|--|---
-'dummyInput' | File | Dummy input to run the workflow
-'exitCode' | Int | Code to fail the workflow as needed
-'n' | Int | Number of lines to log
+#### Required workflow parameters:
+Parameter|Value|Description
+---|---|---
+`dummyInput`|File|File from Vidarr
+
+
+#### Optional workflow parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+`exitCode`|Int|0|Exit code
+`n`|Int|10|Number of lines to log to stderr
+
+
+#### Optional task parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+`log.mem`|Int|1|Memory (in GB) to allocate to the job
+`log.runtime_seconds`|Int|60|The amount of time (in seconds) to simulate processing/sleep for
+`log.timeout`|Int|1|Maximum amount of time (in hours) the task can run for
+
 
 ### Outputs
-Output | Type | Description
----|--|---
-'err' | File | Tail of stderr (with n lines)
 
-### Commands
+Output | Type | Description | Labels
+---|---|---|---
+`stdout`|File|stdout lines produced|vidarr_label: stdout
+`stderr`|File|stderr lines produced|vidarr_label: stderr
+
+
+## Commands
+ This section lists command(s) run by WORKFLOW workflow
+ 
+ * Running WORKFLOW
+ 
+ === Description here ===.
+ 
+ <<<
+     set -euo pipefail
+     # Output n lines to stderr
+     for (( i = 1; i <= ~{n}; i++ )) ; do
+       echo "This is a place holder stdout line ${i}"
+       echo "This is a place holder stderr line ${i}" 1>&2
+     done
+     exit ~{exitCode}
+   >>>
```

Comment:
I don't really understand this section? What is WORKFLOW workflow?

Reply:
This is part of commands.txt that was added. I'll clean this up.